.authors.yml

```diff
@@ -7,7 +7,7 @@
   aliases:
   - Mike Sarahan
   - Michael Sarahan
-  num_commits: 2000
+  num_commits: 4000
```

Comment:
Note, Daniel had used his fixed variant of rever before: https://github.com/regro/rever/pull/266 which hasn't been released yet

Reply:
Since this stat is unused I haven't seen the need to clone and use an unreleased version of rever to get the right number.

However, since you did this for https://github.com/conda/conda/pull/13153 I guess I should just do the same here.

We need to get rever released, amending our release docs to indicate that an unreleased version of rever is necessary is...undesired?

Reply:
I think we should poke the rever maintainers to do a release, we can do that in the conda meeting today

Reply:
Also see this convo from a year ago https://github.com/regro/rever/issues/258

Reply:
See https://github.com/conda/conda-build/pull/5013/commits/67103895029b43f9041f5020f26169dfe5e35837